index.html

```diff
@@ -23,9 +23,30 @@
     <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <style>
+        .sunset-banner {
+            width: 100%;
+            background-color: #e34850;
+            padding: 5px 10px;
+            text-align: center;
+            margin-bottom: 10px;
+            border-radius: 4px;
+            color: #f5f5f5;
+        }
+
+        .sunset-banner a {
+            color: #f5f5f5;
+            text-decoration: underline;
+        }
+    </style>
 </head>
 <body>
     <div class="container">
+        <div class="sunset-banner">
+            <span>The glossary site has moved to a new location and we will be shutting this page down.</span>
```

Comment:
Will be shutting... be verb alert! 
Maybe: "...we will shut down this page soon." (or you can give a dat, like July 1, or not)

Reply:
🐝 removed